client/js/gdpr/chart/index.js

```diff
@@ -0,0 +1,345 @@
+import {SVG_NS} from '../../util.js';
+
+const RUN_SCALE_X = 30;
+const RUN_SCALE_Y = 30;
+const NUM_POSITIONS = 15;
+
+/**
+ * Creates an SVG chart inside of `el` with the given data.
+ *
+ * @param {HTMLElement} container The container element for this chart.
+ * @param {Map<string, number[]>} histories The ranking history for each
+ * track.
+ * @param {Date[]} dates The date of each history entry.
+ */
+export function createChart(container, histories, dates) {
```

Comment:
Remove this function. It's duplicated in your earlier code.

Reply:
Actually, the other file is the duplicate. I don't know why js/chart/index.js still exists, it was supposed to be deleted.